governance/program/src/state/governance.rs

```diff
@@ -100,3 +107,23 @@ pub fn get_account_governance_address<'a>(
     )
     .0
 }
+
+/// Validates governance config
+pub fn assert_is_valid_governance_config(
+    governance_config: &GovernanceConfig,
+    realm_info: &AccountInfo,
+) -> Result<(), ProgramError> {
+    if realm_info.key != &governance_config.realm {
+        return Err(GovernanceError::InvalidGovernanceConfig.into());
+    }
+
+    assert_is_valid_realm(realm_info)?;
+
+    if governance_config.vote_threshold_percentage < 50
```

Comment:
Yeah, I'm addressing this issue in the following PR for voting workflow.
With threshold < 50 we have a race condition where you can vote on `Proposal` and reach the threshold to tip the vote before others can have a chance to vote Nay. 
In the voting PR I'm adding  logic to only tip votes if it's not possible to vote the other way any longer, and in case vote can't be tipped automatically  by the the end of voting period I'm adding a new instruction `ComleteVote` to manually transition `Proposal` `Voting` state to either `Succeeded` or `Defeated` state